### ISSUE TITLE
Energie Ampel API wrong?

### ISSUE THREAD

**Issue by Uatschitchun** [studio-arrenberg/quartiersplattform#153]
When comparing the `Energie Wetter` on QP with Energie Wetter on [WSW site](https://www.wsw-online.de/wsw-digital/smarte-loesungen/wsw-energiewetteruhr/), those differ!?

How comes? Which one is correct? Something wrong with the API?

**Comment by j0hannr** [comment]
How do they differ, in time as a delay or are the completely different?

Do you have screenshots to compare?

**Comment by Uatschitchun** [comment]
Compare for yourself. Those are screenshots from just now (20230402 - 19:40)

![Energieampel - 20230402](https://user-images.githubusercontent.com/25088926/229369621-a48dd421-9dce-4633-a294-bfbc9978ef9f.png)
![Energiewetter 20230402](https://user-images.githubusercontent.com/25088926/229369625-f652c9b3-c5c4-44ce-a5f4-6e8ad5b08fb6.png)

**Comment by Uatschitchun** [comment]
Die Energieampel tut aktuell gar nicht mehr!

**Comment by j0hannr** [comment]
@Uatschitchun Ja, leider übermittelt die WSW Die Daten leider nicht mehr an uns. 

> Wir hatten häufiger Beschwerden, dass Online- und Print Version differieren. Deshalb müssen wir auf die aktuellere und damit genauere Online Version abzielen und auf diese verweisen.
Wir bilden also beispielhaft nur noch einen Tag in der Rundschau ab und verweisen auf  [Energiewetter Wuppertal: Wuppertaler Stadtwerke (wsw-online.de)](https://www.wsw-online.de/zukunft-energie/energiewende/energiewetter-wuppertal/).
Wir hoffen, Sie können unsere Überlegungen nachvollziehen.
WSW

Wir könnten von [dieser Seite](https://energiewetteruhr.wsw-online.de/) die API nutzen und die das Plugin der [Energieampel](https://github.com/studio-arrenberg/energie-ampel-plugin) umschreiben.
